nima/webserver/webserver/src/test/java/io/helidon/nima/webserver/WebServerConfigTest.java

```diff
@@ -148,7 +147,7 @@ void testMediaSupportManualConfigJson() throws IOException {
                 .config(config.get("server"))
                 .mediaContext(MediaContext.builder()
                         .discoverServices(false)
-                        .addMediaSupportProvider(new JsonpMediaSupportProvider())
+                        .addMediaSupport(new JsonpSupport())
```

Comment:
All MediaSupport implementation must have a private/package local/protected constructor only (whatever is the lowest one we need to use).
So you cannot create an instance like this (this should fail compilation). 
Please fix it for all supports, and use `create()` or `create(Config)` methods instead

Reply:
All except String support have private constructors + create factory method.
StringSupport class is being extended in tests so constructor has protected constructor.